extensions/amp-story/1.0/media-pool.js

```diff
@@ -600,33 +595,6 @@ export class MediaPool {
     return swapOutOfDom;
   }
 
-  /**
-   * @param {function(string)} callbackFn
-   * @private
-   */
-  forEachMediaType_(callbackFn) {
-    Object.keys(MediaType).forEach(callbackFn.bind(this));
```

Comment:
The Object.keys() before was causing MediaType to be deopted. It's caught by the linter successfully once renamed to _Enum.

It was rather silly on my part to write this as a loop originally, and it's a smaller bundle size to unroll these loop helpers.